trace/client.go

```diff
@@ -369,29 +474,49 @@ func Flush(cl *Client) error {
 	return <-ch
 }
 
+// FlushError is an aggregate error type indicating that one or more
+// backends failed to flush.
+type FlushError struct {
+	Errors []error
+}
+
+func (fe *FlushError) Error() string {
+	return fmt.Sprintf("Errors encountered flushing backends: %v", fe.Errors)
```

Comment:
It would get printed in logs, and I think we've erred towards printing errors as a single line, even if it sucks for initial readability. Splitting stuff into multiple lines makes it hard to tease apart in log aggregators.